src/lambdas/initiateAthenaQuery/handler.ts

```diff
@@ -22,95 +16,14 @@ export const handler = async (
 
   const zendeskId = retrieveZendeskIdFromEvent(event)
```

Comment:
I would put retrieveZendeskIdFromEvent back in it the handler, and then put the tests for the event structure being malformed here. It doesn't make sense to do that in the other file, and then export it.

Reply:
done

Reply:
done